org.briarproject.desktop.appdata.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.briarproject.desktop</id>
```

Comment:
an app-id that ends in `.desktop` isn't a good idea.

Reply:
I agree. But here we are, in a similar situation as with another app, called "desktop", that I maintain: https://github.com/flathub/chat.delta.desktop/

By now, I think technical reasons have been eliminated. I often get confused, especially when it comes to .desktop files. So it increases maintenance burden. Do you have a suggestion for improvement?

Reply:
I think at least making it uppercase would be saner. Alternatively, `org.briarproject.Briar`.

Reply:
that's a good suggestion! Thanks!